README.md

```diff
@@ -43,31 +42,31 @@ original author of this extension is @siegebell.
 * LtacProf results treeview
 
 ## Requirements
-* VsCode 1.38.0 or more recent
-* Coq 8.7.0 or more recent
+* VsCode or VsCodium 1.38.0, or more recent
+* Coq 8.7.0, or more recent
 
 ## Installation
-The recommended way to install VsCoq is via the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=maximedenes.vscoq) or [Open VSX](https://open-vsx.org/extension/maximedenes/vscoq) .
+The recommended way to install VsCoq is via the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=maximedenes.vscoq) or [Open VSX](https://open-vsx.org/extension/maximedenes/vscoq).
 
 ## Screenshots
 ![vscoq](https://user-images.githubusercontent.com/647105/64025392-dbf12100-cb3c-11e9-8e7f-5c63296500f9.png)
 
 ## Instructions
 1. install [Coq](https://coq.inria.fr/download)
-2. install [vscode](https://code.visualstudio.com/)
-3. run `code`
+2. install [VsCode](https://code.visualstudio.com/) or [VsCodium](https://vscodium.com/)
```

Comment:
I don't `VsCode` is a standard way to write. It seems it should either be `vscode` or `VS Code` (see e.g. https://code.visualstudio.com/blogs/2022/05/18/dev-container-cli or other places on the website where they use `VS Code`).

Reply:
I switched to `VS Code` everywhere, abbreviating `Visual Studio Code`.